src/main/java/org/folio/consortia/repository/SharingRoleRepository.java

```diff
@@ -22,6 +22,10 @@ public interface SharingRoleRepository extends JpaRepository<SharingRoleEntity,
     "WHERE sr.roleId = ?1 AND sr.isCapabilitySetsShared = true")
   Set<String> findTenantsByRoleIdAndSharedCapabilitySets(UUID roleId);
 
+  @Query("SELECT sr.tenantId FROM SharingRoleEntity sr " +
```

Comment:
Perhaps like Serhii said on the previous PR, this can be remove to rely more on Spring Data